torch/_functorch/config.py

```diff
@@ -75,6 +75,9 @@
 # from dynaom in AOTAutogradCache
 autograd_cache_normalize_inputs = not is_fbcode()
 
+# Enable debug mode at first invocation to check if custom ops are valid
+check_custom_op_mode = False
```

Comment:
is this something we could turn on in the ci at least ?

Reply:
Yep i will do for OSS